Estimate Precision Visualization.ipynb

```diff
@@ -179,7 +179,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The below function calculates the probability of the transmitter being located at tx given the drone's location at dx, the signal model parameters n, P\\_rx, P\\_tx, and the standard variation of distance errors D\\_std.  This asumes that the source of error is a normal distribution that is additive to the calculated distances."
+    "The below function calculates the probability of the transmitter being located at $tx$ given the drone's location at $dx$, the signal model parameters $n$, $P\\_rx$, $P\\_tx$, and the standard variation of distance errors $D\\_std$.\n",
+    "\n",
+    "This assumes that the source of error is a normal distribution that is additive to the calculated distances.\n",
+    "\n",
+    "The probability density function is as follows:\n",
+    "\n",
+    "$$f(x) = \\frac{\\exp(\\frac{-x^{2}}{2})}{D\\_std\\sqrt{2\\pi}}$$\n",
+    "\n",
+    "where x is described as\n",
+    "\n",
+    "$$x = \\frac{\\left|\\left|dx - tx\\right|\\right| - \\widehat{d}}{D\\_std}$$\n",
+    "\n",
+    "and $\\widehat{d}$ is equal to\n",
+    "\n",
+    "$$\\widehat{d} = 10 ^ {\\frac{P_{tx} - P_{{rx}_i}}{10n}}$$"
```

Comment:
Please check that your notation is consistent with the notation used earlier in this document.  It seems you have escaped backslashes.

Reply:
fixed the notation. Let me know if there's anything else.

Reply:
Unknown what standard variation is.  Standard deviation of x is normally denoted as $\sigma_x$, variance is denoted as $\sigma^2_x$.  Note that we are using $l_{tx}$ to denote the transmitter location vector, $l_{rx}$ to denote the receiver location vector, and $\left|\vec{x}\right|$ to denote the l2 norm of a vector.

Reply:
Just updated the changes. Simplified the equation a bit, but it's still fairly dense.

Reply:
Notation corrected, using vector and \left\right pairing